packages/google_adsense/lib/src/ad_unit_widget_web.dart

```diff
@@ -77,7 +75,10 @@ class AdUnitWidgetWeb extends AdUnitWidget {
 class _AdUnitWidgetWebState extends State<AdUnitWidgetWeb>
     with AutomaticKeepAliveClientMixin {
   static int adUnitCounter = 0;
-  double adHeight = 1.0;
+  // Make the ad as wide as the available space, so Adsense delivers the best
+  // possible size.
+  Size adSize = const Size(double.infinity, 1);
+  // Size adSize = const Size(600, 1); // It seems ads don't resize, do we need to define fixed sizes for ad units?
```

Comment:
Not sure I follow the point about resizing. AdSense supports both responsive and fixed size ad units. This will impact the size of rendered ad and I expect Widget size to adjust to it. Let's sync on this one

Reply:
The reason why I wanted something like this is that the responsive ads don't seem to be resizing with the app? This might be a problem of having a fixed width in the *Widget* now that I think about it, hmmm....

Reply:
I think you're right on this, only the height of the ad unit needs to be updated. The moment I added an explicit width, I messed up the resizing of the ad units. I'll try to rework this bit today and understand the problem a little bit better.